app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    name_file = command.split(" ")
+    print(command)
+    if name_file[1] != name_file[2] and name_file[0] == "cp":
```

Comment:
The condition checks if the command is 'cp' and if the source and destination files are different. However, there is no feedback or error handling if the command is incorrect or if the files are the same. Consider adding error messages or exceptions for these cases.